turbopack/crates/turbopack-tests/tests/snapshot/imports/ignore-comments/input/index.js

```diff
@@ -11,9 +11,8 @@ import(/* turbopackIgnore: true */ "./ignore.mjs");
 require(/* webpackIgnore: true */ "./ignore.cjs");
 require(/* turbopackIgnore: true */ "./ignore.cjs");
 
-// and for workers
-new Worker(/* webpackIgnore: true */ "./ignore.mjs");
```

Comment:
Is change of `webpackIgnore` to `turbopackIgnore` intentional? What happens if user migrates to `turbopackIgnore` and use webpack for the production build?